nix/sources.json

```diff
@@ -46,5 +46,17 @@
         "type": "tarball",
         "url": "https://github.com/NixOS/nixpkgs/archive/99a3d7a86fce9e9c9f23b3e304d7d2b1270a12b8.tar.gz",
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
+    },
+    "pre-commit": {
```

Comment:
Why is this here?

Reply:
Intent: get lates version of pre-commit which also supporte post-* hooks. (The previously used version didn't.) Pinning to a well known version made the converging install/uninstall bash code less brittle.

Why is it there: due to @roberth 's suggestion as to the means to achieve said intent.

Reply:
Details https://github.com/cachix/pre-commit-hooks.nix/commit/2556038aa1799c08659a1ed47e978bf191774086

Reply:
It doesn't seem to me this is used though? `cfg.package` will still use `pkgs.pre-commit`.

Maybe it we should bump `nixpkgs` to master instead.

Reply:
Oh, ok. I learnd something. Thanks, will do.

Reply:
Done